### PR TITLE
Add KBC schema

### PR DESCRIPTION
**Pull request by tomdaffurn**

### CODE REVIEW

**Comment by frankhinek** [comment]
@tomdaffurn @mistermoe Should we rename this repo to be more generalized if we're going to include both KCC and KBC specs / schemas?  Similar to the [issue](https://github.com/TBD54566975/known-customer-credential/issues/22) @decentralgabe raised.

Another thought is that we could create a `schemas` or `json-schemas` sub-directory in this repo, which would make it simpler for automating hosting.

File: kbc.schema.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "$schema":"https://json-schema.org/draft/2020-12/schema",
```
Comment (decentralgabe) [inline kbc.schema.json:2]: I would prefer we used this version everywhere, just noting that the other schemas use draft 7.

I'd recommend keeping this and let's take an action to update the other schemas.

Comment (decentralgabe) [reply kbc.schema.json:2]: We also need to make sure our libs are set up to handle 2020-12

File: kbc.schema.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  "type":"object",
+  "properties":{
```
Comment (decentralgabe) [inline kbc.schema.json:4]: should we also add `id` as a top level required property?

File: kbc.schema.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "id"
+      ]
+    },
+    "issuer":{
+      "type":"string",
+      "pattern":"^did:[a-z0-9]+:[a-zA-Z0-9.-_:]+$"
+    },
+    "issuanceDate":{
+      "type":"string",
+      "format":"date-time"
```
Comment (decentralgabe) [inline kbc.schema.json:23]: noting that the regex for `date-time` is not quite conformant to xml date time as required by the spec

you can find a full regex for xml date time here https://github.com/w3c/vc-data-model/blob/main/schema/verifiable-credential/verifiable-credential-schema.json#L216

**Comment by tomdaffurn** [comment]
> @tomdaffurn @mistermoe Should we rename this repo to be more generalized if we're going to include both KCC and KBC specs / schemas? Similar to the [issue](https://github.com/TBD54566975/known-customer-credential/issues/22) @decentralgabe raised.
> 
> Another thought is that we could create a `schemas` or `json-schemas` sub-directory in this repo, which would make it simpler for automating hosting.

Moved to a dir 👍  We should have a conversation about where the schemas should live. I'll post about it elsewhere

File: kbc.schema.json (hunk 1 above)
Comment (mistermoe) [reply kbc.schema.json]: lib support for `2020-12` is largely untested at the moment. we know for a fact that 07 works. if we can get someone with cycles to test `2020-12` across several languages or even just rust, js, go, and dart that'd be awesome

**Review by mistermoe — APPROVED** [review]
hesitant about `2020-12` for the reasons explained in the comments i left. aside from that looks good!

**Pull request merged**